src/averageAges.js

```diff
@@ -37,7 +47,12 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(mother => withChildren
+    ? mother.sex === 'f' && people.some(child => child.mother === mother.name)
+    : mother.sex === 'f');
```

Comment:
'f' repeated twice 
if that person is someone's mom, it's definitely a woman

Reply:
U r right) fixed